src/components/10-atoms/button/index.js

```diff
@@ -145,7 +141,7 @@ class AXAButton extends InlineStyles {
     // form behaviour works (submit, reset, etc). The reason why it works with fake button is
     // that fake button is NOT inside a ShadowDOM. The event instead
     // bubbles out of ShadowDOM, hence the stop propagation trick
-    if (eventIsTrusted(e) && isNativeShadowDOM && this.isTypeSubmitOrReset) {
+    if (e.isTrusted && isNativeShadowDOM && this.isTypeSubmitOrReset) {
```

Comment:
```
// IF statement that can be removed with IE11 support dropping
if (document.documentMode && eventIsTrusted(e) && isNativeShadowDOM && this.isTypeSubmitOrReset) {
  // duplicated code, that will be removed with ie11 support fading out
} else if (e.isTrusted && isNativeShadowDOM && this.isTypeSubmitOrReset) {
  //... if-body
}
```

Reply:
done (the submit bug is not relevant for ie11)